From 56e212a0c9af4e5e53c0fa9276b0f6f15b4344b4 Mon Sep 17 00:00:00 2001
From: Thomas Preud'homme <robotux@celest.fr>
Date: Thu, 3 Jan 2013 15:21:53 +0100
Subject: Build ultracopier with resources

Add a .pro file to build ultracopier with plugins and themes.

Origin: vendor
Forwarded: no
Last-Update: 2013-01-03
---
 build.pro |   18 ++++++++++++++++++
 1 file changed, 18 insertions(+)
 create mode 100644 build.pro

diff --git a/build.pro b/build.pro
new file mode 100644
index 0000000..29c617f
--- /dev/null
+++ b/build.pro
@@ -0,0 +1,18 @@
+# Build ultracopier and the right plugins for the host system
+TEMPLATE = subdirs
+SUBDIRS = ultracopier copyengine oxygen clean teracopy sessionloader
+ultracopier.file = ultracopier.pro
+copyengine.file = plugins/CopyEngine/Ultracopier-0.3/copyEngine.pro
+oxygen.file = plugins/Themes/Oxygen/interface.pro
+clean.file = plugins-alternative/Themes/Clean/interface.pro
+teracopy.file = plugins-alternative/Themes/Teracopy/interface.pro
+unix {
+    sessionloader.file = plugins/SessionLoader/KDE4/sessionLoader.pro
+    SUBDIRS += listener
+    listener.file = plugins/Listener/catchcopy-v0002/listener.pro
+}
+win32 {
+    sessionloader.file = plugins/SessionLoader/Windows/sessionLoader.pro
+    SUBDIRS += pluginloader
+    pluginloader.file = plugins/PluginLoader/catchcopy-v0002/pluginLoader.pro
+}
